### PR TITLE
Ds ethiop12new

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Palaeography filled, slight changes.

### CODE REVIEW

File: DayrSuryan/DSEthiop12.xml [hunk 1]
```diff
@@ -571,29 +571,28 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                        <desc>Crude, poorly trained, irregular hand.
-                           Parts of the manuscript were written probably at different times, hence
+                        <desc>Irregular, nearly crude; by a mediocre scribe.
+                           Parts of the text were written probably at different times, hence
                            variations in the hues of the inks, appearance of the
                            script (e.g., slant and thickness of the strokes) and quality of the execution
                            (diminishing after ca. <locus target="#45"/>).
                           The hand seems to be very similar or identical to
-                          <ref type="mss" target="DSEthiop8" corresp="#h1"/>.
+                          <ref type="mss" target="DSEthiop8" corresp="#h1">
```
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:580]: Hand 1 or 2? :)

File: DayrSuryan/DSEthiop12.xml [hunk 2]
```diff
@@ -571,29 +571,28 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         <seg type="ink">Black, violet.</seg>
                         <seg type="rubrication">Beginning and end.</seg>
                         <date notBefore="1800" notAfter="1899">19th century</date>
-                        <desc>Better trained but not very regular hasty hand.
-                          Right-sloping script. Slender rounded
-                          letter shapes. The hand is very similar or identical to
-                   <ref type="mss" target="DSEthiop8" corresp="#h2"/>.
+                        <desc>Regular, somewhat hasty; by a skilled scribe.
+                           The hand is very similar or identical to
+                   <ref type="mss" target="DSEthiop8" corresp="#h2">Hand 1 of DS Ethiop. 8</ref>.
```
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:595]: also h2 and Hand 1 inconsistency

File: DayrSuryan/DSEthiop12.xml (hunk 1 above)
Comment (thea-m) [inline DayrSuryan/DSEthiop12.xml:580]: I think the correct way to refer to a hand in another manuscript would be 
`<ref type="hand" corresp="DSEthiop8#h1">`

Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop12.xml:580]: ok

File: DayrSuryan/DSEthiop12.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply DayrSuryan/DSEthiop12.xml:595]: corrected

**Pull request merged**